test/unit/js/HttpControllerTests.js

```diff
@@ -36,6 +36,7 @@ describe('HttpController', function() {
     })
     this.res = {
       send: sinon.stub(),
```

Comment:
Do you still need the `send` stub?

Reply:
Yes.

<details>

<pre>
  1) HttpController
       getRawDoc
         "before each" hook for "should get the document without the version":
     TypeError: res.send is not a function
      at /app/app/js/HttpController.js:62:20
      at callCallback (node_modules/sinon/lib/sinon/behavior.js:101:25)
      at Object.invoke (node_modules/sinon/lib/sinon/behavior.js:146:27)
      at Object.functionStub (node_modules/sinon/lib/sinon/stub.js:39:43)
      at Function.invoke (node_modules/sinon/lib/sinon/proxy-invoke.js:47:47)
      at Object.functionStub [as getDocLines] (node_modules/sinon/lib/sinon/proxy.js:215:26)
      at Object.getRawDoc (app/js/HttpController.js:54:23)
      at Context.<anonymous> (test/unit/js/HttpControllerTests.js:139:34)

  2) HttpController
       updateDoc
         when the doc body is too large
           "before each" hook for "should return a 413 (too large) response":
     TypeError: res.status(...).send is not a function
      at Object.updateDoc (app/js/HttpController.js:143:23)
      at Context.<anonymous> (test/unit/js/HttpControllerTests.js:419:36)
</pre>
</details>